Sming/SmingCore/Network/UdpConnection.cpp

```diff
@@ -68,37 +56,21 @@ bool UdpConnection::connect(IPAddress ip, uint16_t port)
 void UdpConnection::send(const char* data, int length)
```

Comment:
I would prefer to have the return type of `UdpConnection::send` changed from `void` to `bool`.  `false` when the allocation did not work and `true` otherwise.  
Return type change should be applied also to the `UdpConnection::sendString` , `UdpConnection::sendTo` and `UdpConnection::sendStringTo` methods.

Reply:
Agreed. As `send()` & `sendTo()` are `virtual` it could break existing code, but the compiler would warn so easy to fix. I've also added `bool` return value for `initialize()`.